deployment/clowdapp.yml

```diff
@@ -22,15 +22,23 @@ objects:
       minReplicas: ${{API_REPLICAS}}
       podSpec:
         initContainers:
+        # will this cause a race condition?
           - command:
-            - ./platform-changelog-migration
+            - platform-changelog
+            - migrate
+            image: ${IMAGE}:${IMAGE_TAG}
+            inheritEnv: True
+          - command:
+            - platform-changelog
+            - seed
```

Comment:
Good question about the race condition here.  Maybe you could do something like `platform-changelog migrate && platform-changelog seed`?  This way the seed command only runs if the migrate was successful.

Reply:
Oh yeah, we can do that haha, sounds good!